src/Discovery/DiscoveryDetails/DataDownloadList/DataDownloadList.tsx

```diff
@@ -1,16 +1,25 @@
 import React from 'react';
 import { List } from 'antd';
-import DataDownloadListItem from './Utils/DataDownloadListItem';
+import DataDownloadListItem from './Interfaces/DataDownloadListItem';
 import './DataDownloadList.css';
 import ProcessData from './Utils/ProcessData';
+import ActionButtons from './ActionButtons/ActionButtons';
 
-const DataDownloadList = ({ sourceFieldData }: any) => {
+const DataDownloadList = ({
+  discoveryConfig,
+  resourceInfo,
+  sourceFieldData,
+}) => {
   const data = ProcessData(sourceFieldData);
   if (data.length === 0) {
```

Comment:
By the design https://docs.google.com/presentation/d/1W0ACAiegMY3QfdUt-8ISQoDjKw6yVj9DqLSl42gCmtU if a type of button is configured to show up in this dataDownloadList from portal config, then they should alwyas be visible across all the study details pages. For example, if `enableDownloadManifest` is set to true in config, then for studies that doesn't have a `__manifest` value in its metadata (means `data.length === 0` here), the "Download File Manifest" button should still be visible, but disabled.
In order to do that we need to change the logic here, to let this component renders the `ActionButtons` child even if `data.length === 0`, but don't render the actual `List` of files

Reply:
This has been updated. The conditional for data.length === 0 has been wrapped around only the ```List``` component

```
 <ActionButtons
        isUserLoggedIn={isUserLoggedIn}
        discoveryConfig={discoveryConfig}
        resourceInfo={resourceInfo}
        healLoginNeeded={healLoginNeeded}
      />
      {data.length !== 0 && (
        <List
          itemLayout='horizontal'
          dataSource={data}
          renderItem={(item: DataDownloadListItem) => (
            <List.Item>
              <List.Item.Meta
                title={item.title}
                description={item.description || ''}
              />
            </List.Item>
          )}
          ```